docs/sphinx/targets/python/pasqal.py

```diff
@@ -23,7 +23,11 @@
 os.environ["PASQAL_AUTH_TOKEN"] = str(token)
 
 cudaq.set_target("pasqal",
-                 machine=os.environ.get("PASQAL_MACHINE_TARGET", None))
+                 machine=os.environ.get("PASQAL_MACHINE_TARGET", "EMU_MPS"))
```

Comment:
Now there is a unused env variable in the integration test that should be removed.

Also: for the users EMU_MPS is almost always better (up to ~100 qubits vs 12 for EMU_FREE), but for the CI running EMU_FREE will often be faster as EMU_MPS will by default run on a GPU in our cluster.

Reply:
Ah, I see now the env var is still used. I must have looked at something strange because I thought it was removed.